packages/space-opera/src/redux/initial_environment_images.ts

```diff
@@ -22,11 +22,11 @@ import {EnvironmentImage} from './lighting_state.js';
  */
 export const INITIAL_ENVIRONMENT_IMAGES: EnvironmentImage[] = [
   {
-    uri: './node_modules/@google/model-viewer-shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
+    uri: '/shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
     name: 'Spruit Sunrise'
   },
   {
-    uri: './node_modules/@google/model-viewer-shared-assets/environments/aircraft_workshop_01_1k.hdr',
+    uri: '/shared-assets/environments/aircraft_workshop_01_1k.hdr',
```

Comment:
Sure you don't need a `./` here?

Reply:
nope, this explicitly resolves to the root of the web server.